code/core/package.json

```diff
@@ -437,8 +437,8 @@
     "@storybook/global": "^5.0.0",
     "@testing-library/jest-dom": "^6.6.3",
     "@testing-library/user-event": "^14.6.1",
-    "@vitest/expect": "3.0.9",
-    "@vitest/spy": "3.0.9",
+    "@vitest/expect": "^3.2.4",
+    "@vitest/spy": "^3.2.4",
```

Comment:
We need @kasperpeulen's view on this, because this can affect the exports from `storybook/test`